x-pack/plugins/session_view/public/components/session_view/styles.ts

```diff
@@ -11,22 +11,28 @@ import { CSSObject } from '@emotion/react';
 import { euiLightVars as theme } from '@kbn/ui-theme';
 
 interface StylesDeps {
-  height: string | undefined;
+  height?: string;
+  isFullScreen?: boolean;
 }
 
-export const useStyles = ({ height = '500px' }: StylesDeps) => {
+export const useStyles = ({ height = '500px', isFullScreen }: StylesDeps) => {
   const { euiTheme } = useEuiTheme();
 
   const cached = useMemo(() => {
     const { border } = euiTheme;
 
+    // 118px = Session View Toolbar height + Close Session button height + spacing margin at the bottom
+    const sessionView: CSSObject = {
+      height: `${isFullScreen ? 'calc(100vh - 118px)' : height}`,
```

Comment:
@mitodrummer / @zizhouW any concerns about that?

Reply:
would it make sense to put 118 into a const? and then maybe we can have a test to compare that const height with the actual toolbar height. to avoid any regressions?

Reply:
yeah, I think it makes sense to add that as part of a test for `isFullScreen` for the Session View component, maybe we should do that when we reiterate it again when fixing the remounting bug when toggling fullscreen because it might change the way it works